src/main/reactjs/src/configs/i18n.ts

```diff
@@ -65,6 +66,24 @@ export const getCurrentLang = (): string => {
   return i18n.language;
 };
 
+export const getLanguageTranslated = (
+  language: Language | undefined
+): string => {
+  if (!language) {
+    return '';
+  }
+  switch (getCurrentLang()) {
+    case 'fi-FI':
+      return language.fi;
+    case 'sv-SE':
+      return language.sv;
+    case 'en-GB':
+      return language.en;
+    default:
+      return language.fi;
+  }
+};
+
```

Comment:
Mietin että pitäisikö tämän sijasta kuitenkin laittaa suosiolla koodistosta saadut käännökset i18next-koneistoon? Ts. kun on saatu koodiston kielet backendilta, niin lisättäisiin ne i18nextin saataville (esim. https://www.i18next.com/how-to/add-or-load-translations#add-after-init) jolloin rajapinta käännöksiin pysyisi samana. 

Oletan että jatkossa kuitenkin tarvittaisiin koodistosta muitakin käännöksiä - esim. paikkakuntien nimet - jolloin olisi hyvä olla olemassa yhtenäinen mekanismi käännösten "jälkilataukseen".

Reply:
Kyllä, niiden laittaminen i18next-koneistoon on ihan hyvä idea varsinkin, jos jatkossa tulee lisää käännöksiä sieltä. Frontend voisi ladata niitä aina sovelluksen alustuksen yhteydessä.

Rajapintaa pitäisi muokata myös sen mukaisesti. Se voisi palauttaa esim. tällaisen vastauksen:

{
    'fi-FI': [{"key": "akt.koodisto.jotain, "value": "käännös"} ...]
    'sv-SE': [{"key": "akt.koodisto.jotain, "value": "käännös"} ...]
    'en-GB': [{"key": "akt.koodisto.jotain, "value": "käännös"} ...]
}